src/patternfly/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -160,18 +160,20 @@ import './MenuToggle.css'
 
 ### Plain
 ```hbs
-{{#> menu-toggle menu-toggle--IsPlain="true" menu-toggle--attribute='aria-label="Actions"'}}
+{{> menu-toggle menu-toggle--UsesButton=true menu-toggle--IsPlain=true menu-toggle--IsDiv=true}}
+&nbsp;
+{{#> menu-toggle menu-toggle--IsPlain=true menu-toggle--attribute='aria-label="Actions"'}}
```

Comment:
Do we plan to support both a div wrapper and button wrapper for instances like this? With the div wrapper the focus styling ends up not working correctly, since those styles end up on the `div.pf-ct-c-menu-toggle`. There's still an outline, but the background isn't changing like I'd expect (at least in dark mode).

Reply:
Not both, transitionally we need to support `button.pf-v5-c-menu-toggle__button`, but should be using `.pf-v5-c-button`.